edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html

```diff
@@ -0,0 +1,148 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions message-top-bar" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Awaiting Proctoring Software Installation and Setup
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      You have chosen to take {{display_name}} as a proctored exam. You should be directed to a new window
+      with installation and setup instructions. You can also <a href="{{software_download_url}}" target="_blank">open the installation window directly.</a>
+    {% endblocktrans %}
+  </p>
+  <div class="proctored-exam-message">
+    <h3>
+    {% blocktrans %}
+      Here is your unique exam code. You will be asked for it during the setup.
+    {% endblocktrans %}
+    </h3>
+    <h2> {{exam_code}}<span class='copy-to-clipboard'></span></h2>
+    <p>
+      {% blocktrans %}
+        Do not share this code. It is linked to your {{platform_name}} account and can be used only once.
```

Comment:
wait, if we put "Exam Code" here, then what about the "copy to clipboard" button?